app/models/katello/glue/pulp/repos.rb

```diff
@@ -69,7 +69,7 @@ def sync_status
 
       def sync_summary
         summary = {}
-        last_repo_sync_task_group.each do |task|
+        last_repo_sync_task_by_repoid.values.each do |task|
```

Comment:
@jlsherrill, @chris1984 looks like this also makes the product-index request slow, because `last_repo_sync_task_by_repoid()` is pretty slow.
I will look into making this faster.

Otherwise, I expect that we will have to remove this change again for the release.

Reply:
addressed in https://github.com/Katello/katello/pull/8562